### PR TITLE
add face cropping and size to author image

### PR DESCRIPTION
**Pull request by bennash**
## Problem
On the [/media/authors](https://www.crossroads.net/media/authors/) page, some author photos are being cropped to show female torsos instead of faces. [See attached screenshot](https://rally1.rallydev.com/slm/attachment/632020321209/author%20photo%20cropping.png)

Deliverable requested:

HMW crop those images to faces instead on those pages?

## Solution
Fix imgix params to include face cropping, possible previous typo/mistake

### Corresponding Branch
https://github.com/crdschurch/crds-net/tree/author-image-crop-update

## Testing
screen capture of problem and solution posted to slack channel for crossroads

### CODE REVIEW

File: redirects.csv [hunk 1]
```diff
@@ -1,3 +1,4 @@
+/may1/*,https://www.crossroads.net/content-unavailable/:splat,"302! Country=AD,AE,AF,AG,AI,AL,AM,AO,AQ,AR,AS,AT,AU,AW,AX,AZ,BA,BB,BD,BE,BF,BG,BH,BI,BJ,BL,BM,BN,BO,BQ,BR,BS,BT,BV,BW,BY,BZ,CA,CC,CD,CF,CG,CH,CI,CK,CL,CM,CN,CO,CR,CU,CV,CW,CX,CY,CZ,DE,DJ,DK,DM,DO,DZ,EC,EE,EG,EH,ER,ES,ET,FI,FJ,FK,FM,FO,FR,GA,GB,GD,GE,GF,GG,GH,GI,GL,GM,GN,GP,GQ,GR,GS,GT,GU,GW,GY,HK,HM,HN,HR,HT,HU,ID,IE,IL,IM,IN,IO,IQ,IR,IS,IT,JE,JM,JO,JP,KE,KG,KH,KI,KM,KN,KP,KR,KW,KY,KZ,LA,LB,LC,LI,LK,LR,LS,LT,LU,LV,LY,MA,MC,MD,ME,MF,MG,MH,MK,ML,MM,MN,MO,MP,MQ,MR,MS,MT,MU,MV,MW,MX,MY,MZ,NA,NC,NE,NF,NG,NI,NL,NO,NP,NR,NU,NZ,OM,PA,PE,PF,PG,PH,PK,PL,PM,PN,PR,PS,PT,PW,PY,QA,RE,RO,RS,RU,RW,SA,SB,SC,SD,SE,SG,SH,SI,SJ,SK,SL,SM,SN,SO,SR,SS,ST,SV,SX,SY,SZ,TC,TD,TF,TG,TH,TJ,TK,TL,TM,TN,TO,TR,TT,TV,TW,TZ,UA,UG,UM,UY,UZ,VA,VC,VE,VG,VI,VN,VU,WF,WS,YE,YT,ZA,ZM,ZW"
```
Comment (tcmacdonald) [inline redirects.csv:1]: Is this supposed to be here? Seems unrelated to this PR.

**Review by tcmacdonald — APPROVED** [review]

**Review by bennash — COMMENTED** [review]
removed extra change from previous task

**Review by tcmacdonald — APPROVED** [review]
